builtin_object.go

```diff
@@ -273,6 +273,17 @@ func builtinObjectKeys(call FunctionCall) Value {
 	panic(call.runtime.panicTypeError("Object.Keys is nil"))
 }
 
+func builtinObjectValues(call FunctionCall) Value {
+	if obj, keys := call.Argument(0).object(), []Value(nil); nil != obj {
+		obj.enumerate(false, func(name string) bool {
+			keys = append(keys, obj.get(name))
+			return true
+		})
+		return objectValue(call.runtime.newArrayOf(keys))
+	}
+	panic(call.runtime.panicTypeError("Object.Keys is nil"))
```

Comment:
Looks like a cut and paste of the panic message here.
```suggestion
	panic(call.runtime.panicTypeError("Object.Values is nil"))
```